app/helpers/application_helper.rb

```diff
@@ -1,11 +1,12 @@
 module ApplicationHelper
 
-  def login_url
-    user_omniauth_authorize_url(provider:'twitter')
-  end
-
-  def login_path
-    user_omniauth_authorize_path(provider:'twitter')
+  def handle_for_user(user)
+    auth     = user.first_authorization
+    provider = auth.provider
+    link_to submitted_user_posts_path(user) do
+      icon(provider) + ' ' + icon(:'angle-right') + ' ' +
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.